src/client/components/common/Button/index.tsx

```diff
@@ -14,7 +14,7 @@ const StyledButton = styled.button<{ outline?: boolean }>`
   flex-direction: row;
   align-items: center;
   justify-content: center;
-  padding: 0 1rem;
+  padding: 0 1.6rem;
```

Comment:
I couldn't find a place where the buttons had 16px of paddng in the figma (they are custom widths mostly), but mostly important this component affects all the buttons in the app, and they should not have this amount of padding.

Reply:
I think the current one should be layoutPadding / 2 (8px)

Reply:
I would fix the connectwalletbutton width to a min width, and left default horizontal padding layoutPadding / 2

Reply:
Also remember mobile, since we don't have the designs yet, leaving big paddings will just create more overhead for us to add media queries

Reply:
cool - have reverted for now